tests/RobotFramework/tests/cumulocity/jwt/jwt_request.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO 
+Library    DateTime
+
+Test Tags    theme:c8y    theme:tokens
+Test Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+Retrieve a JWT tokens
+    ${start_time}=    Get Unix Timestamp    
+    Execute Command    tedge mqtt pub c8y/s/uat ''    
+    ${messages}=    Should Have MQTT Messages    c8y/s/dat    maximum=1    date_from=${start_time}
+    Should Contain    ${messages[0]}    71
+
+*** Keywords ***
+Custom Setup
+    ${DEVICE_SN}=    Setup
+    Set Suite Variable    $DEVICE_SN
+    Device Should Exist    ${DEVICE_SN}
+    Stop Service    tedge-mapper-c8y
```

Comment:
Correct me if I am wrong here, but first `Default Setup` runs then `Custom Setup`  then the test.
Since you are stopping the `tedge-mapper-c8y` here before the test starts, how can you retrieve the jwt token? this mapper must be running right.

Reply:
Yeah, Bridge creation is enough to retrieve the jwt token.